daguerre/tests/unit/test_models.py

```diff
@@ -50,3 +52,41 @@ def test_delete_adjusted_images__delete(self):
                           AdjustedImage.objects.get,
                           pk=adjusted2.pk)
         AdjustedImage.objects.get(pk=adjusted1.pk)
+
+
+class AdjustesImageUploadToTestCase(BaseTestCase):
+
+    def setUp(self):
+        self.instance = None
+        self.filename = '7014c0bdbedea0e4f4bf.jpeg'
+
+    def test_upload_to__default_upload_dir(self):
```

Comment:
Ideally these would use mock to check whether the warning was sent and make sure it only pass if it was / wasn't, as appropriate. But I'd be fine merging without that being added to the tests as long as the behavior is correct.